src/server/dbconfig/schema.js

```diff
@@ -0,0 +1,32 @@
+var mongoose = require('mongoose');
+
+var userSchema = new mongoose.Schema({
+  username: String,
+  password: String,
+  restaurants: [{
+    name: String,
+    visits: Number
+  }],
+  locations: [String]
+});
+
+var restaurantSchema = new mongoose.Schema({
+  name: String,
+  rating: String,
+  foodType: String,
```

Comment:
Rather than food types, we might want to just call it tags and make it be an array so we can tag by many things, like chinese, fast food, vegetarian, etc.